business-registry-dashboard/app/enums/affiliation-token.ts

```diff
@@ -0,0 +1,6 @@
+export interface AffiliationToken {
```

Comment:
Figured it'd be nice to create an interface for the token that's being decoded).